tests/system/action/assignment/test_update.py

```diff
@@ -111,7 +111,9 @@ def prepare_voting_phase_test(
         )
 
     def test_update_phase_to_voting(self) -> None:
+        """Also checks that a pre-existing candidate on the list of speakers works."""
         self.prepare_voting_phase_test(3)
+        self.request("speaker.create", {"meeting_user_id": 1, "list_of_speakers_id": 1})
         response = self.request(
```

Comment:
Both must be checked: Putting on list and putting not on list, because already there, I see one test, hmm...